java/src/com/github/bazel_contrib/contrib_rules_jvm/junit5/BUILD.bazel

```diff
@@ -58,11 +58,18 @@ java_library(
         "--release",
         "8",
     ],
+    resource_strip_prefix = package_name(),
+    resources = ["META-INF/services/org.junit.jupiter.api.extension.Extension"],
     deps = [
         ":system-exit-toggle",
         # The only dependencies here are those required to run
         # a junit5 test. We try not to pollute the classpath, so
         # be very careful when adding new deps here.
+        #
+        # junit-jupiter-api is only a compilation dependency and not listed in JUNIT5_DEPS. This is
```

Comment:
Should we add it to `JUNIT5_DEPS`?

Reply:
Adding it could result in errors due to duplicates in `deps` if users declare it themselves. 

Luckily we don't need to add it as it is a runtime dependency of the Jupiter engine, which is always on the classpath.